api/inviteUser/inviteUserUtil.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { expect } from 'chai'
+import { getInviteUserUrl } from './inviteUserUtil'
+
+describe('inviteuser util', () => {
+    it('should getInviteUserUrl', () => {
+        let url = getInviteUserUrl('http://base')
+        expect(url).to.equal('http://base/refdata/external/v1/organisations/users/')
+
+        url = getInviteUserUrl('http://base')
+        expect(url).to.equal('http://base/refdata/external/v1/organisations/users/')
```

Comment:
duplicate code from line 6 and 7

Reply:
Would not worry about it in spec files. But, good point